rosetta/utils/te_pax_t5x_ckpt_converter/converter/main.py

```diff
@@ -109,6 +108,17 @@ def parse_args():
                         default=False,
                         help="indicate if skip the conversion for LayerNorm.")
 
+    parser.add_argument('--gen-fp8-meta',
+                        action="store_true",
+                        default=False,
+                        help="indicate if generate corresponding FP8 meta."
+                        " Only works when --direction=fw2te")
+    parser.add_argument(
+        '--amax-history-len',
+        type=int,
+        default=1,
```

Comment:
Should we set this to the recommended default of 1024?

Reply:
Done